packages/core/src/themes/buildFont.ts

```diff
@@ -44,7 +44,7 @@ export default function buildFont(fontFamily: string): Theme['font'] {
     textRegular: {
       fontFamily,
       fontSize: 15,
-      lineHeight: 20 / 15,
```

Comment:
before https://github.com/airbnb/lunar/pull/140/files , this value was `22.5px`. bumping back up to `22px`